### PR TITLE
Fixes #105: Trim values read from CSS custom properties.

### LINKED ISSUE #105: Ripple not shown when using native CSS custom properties.
### Description
The ink ripple that usually appears when the element is clicked or focused has the wrong size when native CSS custom properties are enabled.

### Live Demo
http://jsbin.com/walerew/1/edit?html,output

### Browsers Affected

- [x] Chrome
- [x] Firefox
- [ ] Safari 10
- [ ] Edge
- [ ] IE 11

### PR DESCRIPTION
**Pull request by bicknellr**

### CODE REVIEW

File: paper-radio-button.html [hunk 1]
```diff
@@ -215,10 +215,10 @@
       },
 
       attached: function() {
-        var inkSize = this.getComputedStyleValue('--calculated-paper-radio-button-ink-size');
+        var inkSize = this.getComputedStyleValue('--calculated-paper-radio-button-ink-size').trim();
```
Comment (notwaldorf) [inline paper-radio-button.html:218]: 💀

File: test/index.html [hunk 2]
```diff
@@ -16,7 +16,8 @@
   <script>
     WCT.loadSuites([
       'basic.html',
-      'basic.html?dom=shadow'
+      'basic.html?dom=shadow',
+      'basic.html?dom=shadow&lazyRegister=true&useNativeCSSProperties=true'
```
Comment (bicknellr) [inline test/index.html:23]: The whole set of these seems necessary to get Polymer to use native CSS custom properties: https://github.com/Polymer/polymer/blob/ff6e884ef4f309d41491333860a8bc9c2f178696/src/lib/settings.html#L55.

File: paper-radio-button.html [hunk 3]
```diff
@@ -215,10 +215,10 @@
...
         // If unset, compute and set the default `--paper-radio-button-ink-size`.
         if (inkSize === '-1px') {
-          var size = parseFloat(this.getComputedStyleValue('--calculated-paper-radio-button-size'));
+          var size = parseFloat(this.getComputedStyleValue('--calculated-paper-radio-button-size').trim());
```
Comment (notwaldorf) [inline paper-radio-button.html:221]: ☠️

File: test/index.html (hunk 2 above)
Comment (notwaldorf) [inline test/index.html:23]: hey what happens in 1.0 with these flags?

**Review by notwaldorf — APPROVED** [review]
LT(tragic)TM, but 👍

File: test/index.html (hunk 2 above)
Comment (bicknellr) [reply test/index.html:23]: I really only care about `useNativeCSSProperties` here, but the other two are necessary to enable that flag. (link in comment above) :shrug:

Comment (bicknellr) [reply test/index.html:23]: ok, i added a comment about the flags

**Pull request merged**